AT-COMMUNITY-CODE-OF-CONDUCT.md

```diff
@@ -1,5 +1,13 @@
 # Algeria Tech Community Slack - Code of Conduct
 
+## Disclaimer : 
+
+**The Algeria Tech Community is an Algerian Tech only community, means that all the Algerian/Foriegn people in local and abroad are welcome to join, while keeping on respecting all sort of marks/tradition that goes with the Algerian Identity and that sum all the given aspects of that Identity, if the organization team felt that someone, some group are going against that rule, they will be expeled from the channels as a whole.**
```

Comment:
je pense que la phrase, prete à confusion: je dirait plus, "means that all people living in Algeria and all Algerians living abroad are wecome ..."

La manière don't c'est écrit mnt, je comprend que meme "foreign people abroad" are welcome